test/test_block_genesis.py

```diff
@@ -0,0 +1,55 @@
+"""
+Test genesis block
+"""
+
+import pytest
+
+from .util import get_block
+
+FEE_TOKEN_CLASS_HASH = "0x6a22bf63c7bc07effa39a25dfbd21523d211db0100a0afd054d172b81840eaf"
+UDC_CLASS_HASH = "0x7b3e05f48f0c69e4a65ce5e076a66271a527aff2c34ce1083ec6e1526997a69"
+CHARGEABLE_ACCOUNT_CLASS_HASH = "0x4d07e40e93398ed3c76981e72dd1fd22557a78ce36c0515f679e27f0bb5bc5f"
+STARKNET_CLI_ACCOUN_CLASS_HASH = "0x6f500f527355dfdb8093c7fe46e6f73c96a867392b49fa4157a757538928539"
+
+@pytest.fixture(name="last_tx_hash")
+def fixture_last_tx_hash(request):
+    """
+    Fixture for last tx_hash
+    """
+    return request.param
+
+@pytest.mark.usefixtures("run_devnet_in_background")
+@pytest.mark.parametrize(
+    "run_devnet_in_background, last_tx_hash",
+    [
+        (
+            ["--seed", "42", "--accounts", "1"],
```

Comment:
Can't it be any seed? I.e. can't we rely on devnet to generate its own seed when not provided with one instead of having to hardcode something.

Reply:
Actually, this seed can be removed! Fixed in 30b94bbeea5737765c8843bac5cda98707d0adb5